contents/tutorials/event-tracking-guide.md

```diff
@@ -2,184 +2,303 @@
 
-#### 2. Combining multiple events into one action**
+To set up custom events, first, you need to install the library for the language or framework you are using. We have built libraries for a range of languages and frameworks including Javascript, Python, iOS, Android, and more. You can find our [full list of libraries here](https://posthog.com/docs/integrate#libraries).
 
-Let's say you have multiple sign up buttons spread across your website. For some types of analysis it could be useful to know which specific button was clicked, however, it might also be the case that you just care that the user signed up, irrespective of what they clicked to do so.
+For example, with Python (and Python frameworks like Django or Flask), setting up custom events starts with installing the PostHog package with pip (or your package manager of choice). 
 
-Actions fit this use-case perfectly, since events are combined using `OR` operations. That means that if an action is composed of multiple events, it will be registered if _any of the events composing it_ are triggered. 
+```bash
+pip install posthog
+```
 
-In other words, using the example from above, if you have an action called "Sign Up" that targets the clicks on all your sign up buttons, if your user clicks any of the buttons, the action will be registered.
+Second, in whatever library you’ve chosen, you’ll need to configure PostHog. This includes adding your project’s API key and setting the PostHog host.
 
-> **Note:** Actions also work retroactively, meaning that they also tag events that happened in the past, not only events that happened after the action was created.
+```python
+import posthog
 
-### Creating Actions
+# Substitutes posthog.api_key which still exists but has been deprecated
+posthog.project_api_key = '<ph_project_api_key>'
 
-There are many ways to create actions in PostHog. These are:
+# Only necessary if you want to use feature flags
+posthog.personal_api_key = '<ph_personal_api_key>'
 
-**1. Through our toolbar (recommended for actions based on frontend elements)**
+# You can remove this line if you're using app.posthog.com
+posthog.host = '<ph_instance_address>'
+```
 
-To learn how to create actions using our toolbar, you can visit our [dedicated tutorial](/docs/tutorials/toolbar/#creating-actions).
+Third, once the library is installed and configured, events can be captured by calling library methods with user details, event types, and properties. Each of our libraries contains the relevant structure for connecting the arguments to these categories. For example, in Python a capture method call might look like this:
 
-**2. From the events table**
+```python
+def movie_played(movie):
+	movie.view_count += 1
```

Comment:
nit: this looks like "analytics" so it might be confusing to people as they might think this has something to do with PostHog. Aware that the idea here is to show how PostHog integrates with an existing codebase, but just thought I'd mention

Reply:
Good point, I'll just remove it.